x-pack/plugins/maps/public/kibana_services.ts

```diff
@@ -81,3 +82,12 @@ export const getProxyElasticMapsServiceInMaps = () =>
   getKibanaCommonConfig().proxyElasticMapsServiceInMaps;
 export const getRegionmapLayers = () => _.get(getKibanaCommonConfig(), 'regionmap.layers', []);
 export const getTilemap = () => _.get(getKibanaCommonConfig(), 'tilemap', []);
+
+let coreApplication: ApplicationStart;
```

Comment:
Instead of storing and setting `coreApplication` and `coreShareService` separately, can you just update the getters to return `core.application` and `plugins.share`?

Reply:
Sure @nreese